docs/lesson/edit.rst

```diff
@@ -0,0 +1,207 @@
+
+.. figure:: /images/edit_content.png
+   :alt: Editing content
+   :scale: 35 %
+
+   *Fig. 1*
+
+3. To delete the entire exploration, go to the |settings| **Settings** button in the navigation bar, and click the |deleteexploration| button located at the bottom of the page.
+
+.. |deleteexploration| image:: /images/delete_exploration.png
+                       :alt: Delete Exploration button
+                       :scale: 35 %
+
+.. _revert:
+
+Working with older versions of the exploration
+**************************************************
+
+1. Every time you save your exploration, Oppia keeps a record of that version. To save a draft of your exploration, click the |savedraft| button in the top navigation bar. In the pop-up box that appears, you can type in an optional message describing the changes made.
+
+.. |savedraft| image:: /images/save_draft.png
+               :scale: 40 %
+               :alt: Save draft button
+
+.. figure:: /images/save_description.png
+   :alt: Save Draft box
+   :scale: 40 %
+
+   *Fig. 2*
+
+2. You can view a list of all saved versions at any time by clicking the |history| **History** button.
+
+.. |history| image:: /images/history_button.png
+             :scale: 20 %
+             :alt: History button
+
+.. figure:: /images/history.png
+   :alt: History of saved explorations
+
+   *Fig. 3*
+
+From the **History** screen, you have the following options:
+
+.. topic:: Compare selected revisions
+
+   Select any two previous versions and click the |compare| button to view what was changed.
+
+   .. |compare| image:: /images/compare_revisions_button.png
+                :scale: 40 %
+                :alt: Compare selected revisions button
+
+   .. figure:: /images/compare_versions.png
+      :alt: Comparing two versions
+      :scale: 30 %
+
+      *Fig. 4*
+
+.. topic:: Revert to a previously saved version
+
+   Click the *Revert* link next to any version of your exploration to revert and go back to using that version. This change will now be recorded and added to your **History** under the list of changes.  
+
+   .. warning:: When you revert to an older version, you will lose any unsaved changes in your exploration. If you think you might want to come back to your current version at some point later on, make sure you save your draft before reverting so that Oppia has a record of it. 
+
+.. topic:: Download a version
+
+   Click the *Download* link next to a version to download a zip file that will contain all the data for your exploration. It will include a YAML_ file as well as any additional asset files (such as images) that are part of your exploration. You can now share this file with other people if you wish.
+
+   .. _YAML: https://en.wikipedia.org/wiki/YAML
+
+   .. figure:: /images/revert_download.png
+      :alt: Reverting and downloading past verions
+
+      *Fig. 5*
+
+.. _reviewerror:
+
+Reviewing errors and ending the exploration
```

Comment:
This section title sounds quite general, but it looks like it only treats one type of error. Just wanted to check ... is that intentional?

Reply:
Actually, on hindsight it might be better to change the section title to 'Resolving warnings...' I gave a couple of examples of what type of warnings a creator might see. I'll make the change in the next PR.

Reply:
@seanlip, Can we merge this PR now?